src/main/java/io/nats/client/impl/NatsJetStreamImplBase.java

```diff
@@ -61,16 +64,34 @@ ConsumerInfo _getConsumerInfo(String streamName, String consumer) throws IOExcep
     }
 
     ConsumerInfo _createConsumer(String streamName, ConsumerConfiguration config) throws IOException, JetStreamApiException {
```

Comment:
THIS IS THE MOST IMPORTANT CHANGE. It's the primary functionality. For context you could look at the JS code found here: https://github.com/nats-io/nats.deno/pull/353/files#diff-42f288baec60b087dbeb2c90ad843507c2876286e0c445ea8d71dc5908e9a5aeR63